java/jlog.java

```diff
@@ -187,6 +187,20 @@ public Id advance(Id what, Interval interval)
   }
 
   static {
-    System.loadLibrary("jlog");
+    try {
+      System.loadLibrary("jlog");
+   } catch (Exception e) {
+     System.err.println("Cannot load jlog library: " + e);
+     System.exit(-1);
+   }
+    try {
+      // System.load("/opt/circonus/lib/java/libjlog.jnilib");
+      System.loadLibrary("jnijlog");
+   } catch (Exception e) {
+     System.err.println("Cannot load jlog JNI library: " + e);
+     System.exit(-1);
+   }
+
+   System.out.println("Dependant libraries loaded successfully");
```

Comment:
This unconditional print for all jlog consumers isn't going to be well received.  Drop that.